lib/pipefy_message/consumer.rb

```diff
@@ -80,12 +80,23 @@ def process_message
           logger.info(log_context({
```

Comment:
**suggestion (non-blocking):** I know these nested methods call have around for a while, but it would be nice if this could be just a single method call. I am affraid of forgetting to do the correct nesting in the future.

Reply:
I agree; I added this in a previous MR because I honestly couldn't figure out a way to pass those automatically, so that the person writing the code doesn't have to do anything... See [this](https://github.com/pipefy/pipefy_message/pull/33#discussion_r934787557) thread from the CID MR.

Perhaps the best solution would simply be to [delegate logging to the actual app logger](https://pipefy.slack.com/archives/C03NBL5L7RQ/p1660154302953629) (the core's, in our case), which already includes the CID, but then we'd have to figure out a) how to do that and b) how to ensure that the CID has been properly set before we log anything after receiving a message.

Reply:
We can do it later... step by step...